Mathlib/RingTheory/PowerSeries/Basic.lean

```diff
@@ -1832,9 +1836,63 @@ theorem trunc_add (n) (φ ψ : PowerSeries R) : trunc n (φ + ψ) = trunc n φ +
     · rw [zero_add]
 #align power_series.trunc_add PowerSeries.trunc_add
 
+/--
+`R⟦X⟧` is notation for `PowerSeries R`,
+the semiring of formal power series in one variable over a semiring `R`.
+-/
+scoped notation:9000 R "⟦X⟧" => PowerSeries R
```

Comment:
I would put this somewhere nearer the top of the file, and the the definition (and then maybe also use it throughout the file / library!)

Reply:
I'm not suggesting that you change the whole file in this PR, but I would at least move it closer to the definition here

Reply:
Thanks, I did wonder about that. I'll do what you suggest.

Reply:
Looks great! And the diff isn't too big even, thanks for that